src/Categories/Functor/Construction/FreeCategory.agda

```diff
@@ -0,0 +1,65 @@
+{-# OPTIONS --without-K --safe #-}
+
+module Categories.Functor.Construction.FreeCategory where
```

Comment:
Here it's called `FreeCategory` but the construction of the category itself is called `PathCategory`. These should be made consistent.

Reply:
this name is somewhat misleading. it's a functor, not a category, so `FreeCategory` is not quite accurate. not sure what's a better name though. what about `FreeCatF`? at least it makes it clear it's a functor.

Reply:
After a lot of thinking, I went with `PathsOf` as the name of the functor.